apps/test/netsim/NetSimConnection.js

```diff
@@ -1,87 +0,0 @@
-'use strict';
-/* global describe */
-/* global beforeEach */
-/* global it */
-
-var testUtils = require('../util/testUtils');
-testUtils.setupLocale('netsim');
-var assert = testUtils.assert;
-var assertEqual = testUtils.assertEqual;
-var netsimTestUtils = require('../util/netsimTestUtils');
-var fakeShard = netsimTestUtils.fakeShard;
-var assertTableSize = netsimTestUtils.assertTableSize;
-
-var NetSimLogger = testUtils.requireWithGlobalsCheckBuildFolder('netsim/NetSimLogger');
-var NetSimConnection = testUtils.requireWithGlobalsCheckBuildFolder('netsim/NetSimConnection');
-var NetSimClientNode = testUtils.requireWithGlobalsCheckBuildFolder('netsim/NetSimClientNode');
-var NetSimRouterNode = testUtils.requireWithGlobalsCheckBuildFolder('netsim/NetSimRouterNode');
-
-describe("NetSimConnection", function () {
```

Comment:
No, they don't; the only method being tested here was the `getAllNodes` method, which was redundant with `netsimNodeFactory.nodesFromRows` so I removed it, and since I subsequently removed the NetSimConnection type altogether I dropped this test file.